R/colour_nhsbsa.R

```diff
@@ -1,10 +1,50 @@
+#' Check input colour is correct built-in R recognisable
+#' color names or HEX format
+#'
+#' @param x String type of colour palettes
+#'
+#' @return boolean
+#' @export
+#'
+#' @examples
+#' nhsbsaR::is_color(c("red", "#FFFFFF"))
```

Comment:
No need to namespace with `nhsbsaR::`. R CMD check will run the examples to ensure they work, and will have the package loaded to do so.